### PR TITLE
Add guideline for figure captions (close #158)

### LINKED ISSUE #158: Do away with, or better define, "Figure x" convention
**Describe the issue**

By the convention of numbering figures, any time an image is added to a section that does not come at the end of the existing text, every figure that succeeds the new image must have its caption renamed. This is already the case, as an image that was added in section 2 doesn't yet have a "Figure x" caption. To add one and keep up the convention, we'd have to change the captions of all remaining figures in the book.

This convention is already broken in some sections; 4.2.1 uses "Figure 4.2.1-x", yet in 4.2.3 there is a "Figure 10."

I propose either doing away with the "Figure x" convention altogether, and descriptively captioning images without them; or standardizing the section-specific "Figure x.x.x-x" convention, which is more easily maintained.

### PR DESCRIPTION
**Pull request by victoriadrake**
This PR covers issue #<issue number>.

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Adds guidance for future contributors regarding figure captions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ You can as well open up a post on our [Google Group](https://groups.google.com/a
 - Please do use Title Caps for headings, using Title Capitalization as defined by the 'Chicago Manual of Style'. For quick reference you can use this online tool: [https://capitalizemytitle.com/#Chicago](https://capitalizemytitle.com/#Chicago) (make sure you select the "Chicago" tab).
 - Please do use serial or [Oxford commas](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/).
 - Don't use `and/or`, chances are you can simply write `or`. (Note: The OR allows for the same True result as an AND, while also allowing for other combinations producing True results.) Unless, you actually mean something like "A and/exclusive or B" in which case read the sentence to yourself with those words and then figure out a different way to write it :smile:.
+- Caption figures using title case and with the section and sub-sections, for example: *Figure 4.2.1-1: Google Site Operation Search Result Example*.
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply README.md]: Ya that probably makes sense. Let me see if I can put together an ASCII example.

**Comment by victoriadrake** [comment]
Incorporated suggestions from above, and addressed the folder structure to show where the images go. Also minor fixes I noticed.

File: README.md [hunk 2]
```diff
@@ -21,16 +24,53 @@ Despite us being technical, we love having technical and casual chats with other
 - Join [OWASP Slack](https://join.slack.com/t/owasp/shared_invite/enQtNjExMTc3MTg0MzU4LTViMDg1MmJiMzMwZGUxZjgxZWQ1MTE0NTBlOTBhNjhhZDIzZTZiNmEwOTJlYjdkMzAxMGVhNDkwNDNiNjZiOWQ).
 - Join this project's [channel: #testing-guide](https://app.slack.com/client/T04T40NHX/CJ2QDHLRJ) (yes, you can join other channels, we won't stop you!).
 
-Before you start contributing, please check our [**contribution guide**](CONTRIBUTING.md) which should help you get started and follow our best practices.
+Feel free to ask questions, bounce ideas off us, or just hang out and chat!
```
Comment (ThunderSon) [inline README.md]: shouldn't it be `off of us`?

File: README.md [hunk 3]
```diff
@@ -21,16 +24,53 @@ Despite us being technical, we love having technical and casual chats with other
...
 
-You can as well open up a post on our [Google Group](https://groups.google.com/a/owasp.org/forum/#!forum/testing-guide-project)!
+You can also open up a post on our [Google Group](https://groups.google.com/a/owasp.org/forum/#!forum/testing-guide-project)!
 
 ## Style Guidelines
 
 - Please don't write in the first person (Ex: no "I" or "Me" statements).
-- Please do use Title Caps for headings, using Title Capitalization as defined by the 'Chicago Manual of Style'. For quick reference you can use this online tool: [https://capitalizemytitle.com/#Chicago](https://capitalizemytitle.com/#Chicago) (make sure you select the "Chicago" tab).
+- Please do use Title Caps for headings, using Title Capitalization as defined by the *Chicago Manual of Style*. For quick reference you can use this online tool: [https://capitalizemytitle.com/#Chicago](https://capitalizemytitle.com/#Chicago) (make sure you select the "Chicago" tab).
```
Comment (ThunderSon) [inline README.md]: What do you think about this?
```suggestion
- Please do use Title Caps for headings, using Title Capitalization as defined by the *Chicago Manual of Style*. For quick reference you can use this [online tool](https://capitalizemytitle.com/#Chicago) (make sure you select the "Chicago" tab).
```

**Pull request merged**